_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
```

Comment:
I always assumed the origin of the name was "taps into the Merkle root", but I don't actually know what @gmaxwell's thinking was.

Reply:
I originally had to look the word up; but I took it as the key path being the "taproot" because that's the tasty one that you make carrot soup out of, and the merklized scripts would be the other lesser roots that you hope to ignore. The BIP-116/117 approach didn't have a "root" to tap into per se, and the original proposal was pretty explicitly compatible with that I think.